doc/api/cli.md

```diff
@@ -296,6 +296,30 @@ are linked from more than one location in the dependency tree (Node.js would
 see those as two separate modules and would attempt to load the module multiple
 times, causing an exception to be thrown).
 
+Note also that the `--preserve-symlinks` flag does not apply to the main
+module - this special case helps
+`node --preserve-symlinks node_module/.bin/<foo>` work.  If you want the same
+behavior on the main module, also use `--preserve-symlinks-main`.
```

Comment:
Please avoid using `you` in the docs... this can be reworded to:

```
The `--preserve-symlinks` flag does not apply to the main module, allowing
`node --preserve-symlinks node_module/.bin/<foo>` to work. To apply the same
behavior for the main module, also use `--preserve-symlinks-main`.
```

Reply:
thanks for the advise.  I knew second person wasn't ideal, just wasn't sure how to better phrase it.